files/es/web/api/dragevent/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: DragEvent
+slug: Web/API/DragEvent
```

Comment:
@sergiogval I'm not sure what doubts you were having about the slug, but this looks good :) There's also no need for an `original_slug` in this situation because the slug wouldn't really be different if you "translate" it to Spanish.

Reply:
Great, I understand now. Also I got rid of the line regarding the **original_slug**